main.go

```diff
@@ -198,13 +204,20 @@ func run(c *cli.Context) error {
 		return err
 	}
 
+	flushage, err := strconv.Atoi(c.String("flush_age"))
+
+	if err != nil {
+		return err
+	}
+
 	p := &Plugin{
 		Filename:     filename,
 		Path:         path,
 		FallbackPath: fallbackPath,
 		FlushPath:    flushPath,
 		Mode:         mode,
```

Comment:
I think this should also be usable as a command line application for controlling the cache. So I think it should be a separate mode.